components/footer/footer.scss

```diff
@@ -4,3 +4,16 @@ footer {
   bottom: 0;
   left: 0;
 }
+
+.site-footer {
+  background-color: $black;
+
+  .link-list {
+    @media (min-width: $bp-md) {
+      -webkit-column-count: 2;
```

Comment:
We have `autoprefixer` npm task set up so we don't have to add vendor prefix ourselves... unless... that task is no longer working? 🙄

Reply:
nope, sorry force of habit—I'll update that